### PR TITLE
[E2E] Module tests are added

### PR DESCRIPTION
**Pull request by elenaNaboko**
The following tests are added:
- Verify that user can see DB modules on DB list page for Standalone DB.
- Verify that user can see +N icon (where N>1) on DB list page when modules icons don't fit the Module column width.
- Verify that user can hover over the module icons and see tooltip with all modules name.
- Verify that user can hover over the module icons and see tooltip with version.
- Verify that user can see full module list in the Edit mode.

### CODE REVIEW

File: tests/e2e/pageObjects/add-redis-database-page.ts [hunk 1]
```diff
@@ -7,176 +7,242 @@ export class AddRedisDatabasePage {
...
+                paste: true
+            })
+            .typeText(this.portInput, parameters.port, {
+                replace: true,
+                paste: true
+            })
+            .typeText(this.databaseAliasInput, parameters.databaseName, {
+                replace: true,
+                paste: true
+            })
```
Comment (ofersteinberg) [inline tests/e2e/pageObjects/add-redis-database-page.ts:86]: Whi you did it? please revert

File: tests/e2e/pageObjects/add-redis-database-page.ts [hunk 2]
```diff
@@ -7,176 +7,242 @@ export class AddRedisDatabasePage {
...
+     */
+    async addLogicalRedisDatabase(parameters: AddNewDatabaseParameters, index: string): Promise<void> {
+        await t
+            .click(this.addDatabaseButton)
+            .click(this.addDatabaseManually)
+        await t
+            .typeText(this.hostInput, parameters.host, {
+                replace: true,
+                paste: true
+            })
```
Comment (ofersteinberg) [inline tests/e2e/pageObjects/add-redis-database-page.ts:114]: Same here

File: tests/e2e/pageObjects/my-redis-databases-page.ts [hunk 3]
```diff
@@ -7,117 +7,151 @@ export class MyRedisDatabasePage {
...
-      else if (count === 1) {
-          await t.click(this.deleteDatabaseButton);
-          await t.click(this.confirmDeleteButton);
-      }
-      if (await this.toastCloseButton.exists) {
-          await t.click(this.toastCloseButton);
-      }
-  }
+    //Delete database by Name
+    async deleteDatabaseByName(dbName: string): Promise<void> {
```
Comment (ofersteinberg) [inline tests/e2e/pageObjects/my-redis-databases-page.ts:119]: Add parameter description

File: tests/e2e/tests/critical-path/database/modules.e2e.ts [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        //Verify that user can see +N icon (where N>1) on DB list page when modules icons don't fit the Module column width
+        await t.expect(myRedisDatabasePage.moduleQuantifier.textContent).eql('+4');
+        await t.expect(myRedisDatabasePage.moduleQuantifier.exists).ok('Quantifier icon');
+        //Verify that user can hover over the module icons and see tooltip with all modules name
+        await t.hover(myRedisDatabasePage.moduleQuantifier);
+        await t.expect(myRedisDatabasePage.moduleTooltip.visible).ok('Module tooltip');
+        //Verify that user can hover over the module icons and see tooltip with version.
+        await myRedisDatabasePage.checkModulesInTooltip(moduleList);
+    });
+test.skip
```
Comment (ofersteinberg) [inline tests/e2e/tests/critical-path/database/modules.e2e.ts:42]: Please add the reason

File: tests/e2e/pageObjects/add-redis-database-page.ts (hunk 1 above)
Comment (elenaNaboko) [reply tests/e2e/pageObjects/add-redis-database-page.ts]: It was added automatically by IDE when I corrected all spaces in the file. Fixed.

File: tests/e2e/pageObjects/my-redis-databases-page.ts (hunk 3 above)
Comment (elenaNaboko) [reply tests/e2e/pageObjects/my-redis-databases-page.ts:119]: Added

File: tests/e2e/tests/critical-path/database/modules.e2e.ts (hunk 4 above)
Comment (elenaNaboko) [reply tests/e2e/tests/critical-path/database/modules.e2e.ts]: Edit form was not ready from dev perspective. Corrected now and remove .skip label

**Review by ofersteinberg — APPROVED** [review]

**Pull request merged**